### PR TITLE
Tweaks to 0216-qualified-do.rst

### PR DESCRIPTION
**Pull request by ggreif**

### CODE REVIEW

File: proposals/0216-qualified-do.rst [hunk 1]
```diff
@@ -316,7 +316,7 @@ raise an error *even if there is a value for ``(>>=)`` in scope*.
 
 Enabling ``-XQualifiedDo`` doesn't change the meaning of existing do-expressions.
 
-When both ``-XQualifiedDo`` and ``-XRebindableSyntax`` are enabled, ``-XQualifiedDo`` only affects qualified ``do``'s and ``-XRebindableSyntax`` affects the unqualified ``do``'s.
+When both ``-XQualifiedDo`` and ``-XRebindableSyntax`` are enabled, ``-XQualifiedDo`` only affects qualified ``do``s and ``-XRebindableSyntax`` affects the unqualified ``do``s.
```
Comment (ggreif) [inline proposals/0216-qualified-do.rst:319]: Not sure if this renders right, do we need a zero-width space here?

Comment (Ericson2314) [reply proposals/0216-qualified-do.rst:319]: An escaped space `\ ` works.

Comment (Ericson2314) [reply proposals/0216-qualified-do.rst:319]: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#escaping-mechanism

**Comment by ggreif** [comment]
@Ericson2314 please squash on merge!

**Pull request merged**